app/components/timesheet-actions.js

```diff
@@ -1,22 +1,32 @@
 import Component from '@glimmer/component';
-import { task } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import constants from 'frontend-timekeeper/constants';
+const { TIMESHEET_STATUSES } = constants;
 
 export default class TimesheetActionsComponent extends Component {
-  get canSubmitHolidays() {
-    return !this.args.timesheet
-      || this.args.timesheet.isDraft;
-  }
+  @tracked absenceComplete = false;
+  @tracked timesheetComplete = false;
 
-  get canSubmitTimesheet() {
-    return !this.canSubmitHolidays
-      && this.args.timesheet.isAbsenceSubmitted;
+  @action
+  onAbsenceClick(checked) {
```

Comment:
Action handlers are typically named with an active verb like `clickAbsence`.

Reply:
fixed in [38b50ea](https://github.com/redpencilio/frontend-timekeeper/pull/4/commits/38b50eaaf61b2195024c3afdce974fef5d636e19)